int8 radar_status
int8 target_sum
